Provide examples of sensitive information

adborden opened the pull request:
https://github.com/18F/aws-admin/issues/92

After clarifying some questions with GSA IT Security, document some examples and
explicit guidance for handling specific types of sensitive information.

[Preview](https://federalist-cf8341ad-ca07-488c-bd96-0738014c79ca.app.cloud.gov/preview/18f/handbook/feature/sensitive-information/sensitive-information/)

Commit:
Provide examples of sensitive information

After clarifying [some questions][1] with GSA IT Security, document some
examples and explicit guidance for handling specific types of sensitive
information.

[1]: https://github.com/18F/aws-admin/issues/92

JJediny reviewed (approved):
Based on discussion LGTMerge

Commit:
Update sensitive information

Merge content from the [Open Source Policy practices guide][1].

[1]: https://github.com/18F/open-source-policy/blob/master/practice.md#protecting-sensitive-information

adborden committed:
Merge 17c8ce811ce08c59f47614585d5878763ac27f51 into 9681a3653e5e27038c92a9b1ff11a7cdc9204b0f

Commit:
Format code
